metron-platform/metron-common/src/main/java/org/apache/metron/common/dsl/functions/resolver/ClasspathFunctionResolver.java

```diff
@@ -65,24 +72,48 @@
  *
  */
 public class ClasspathFunctionResolver extends BaseFunctionResolver {
-
-  /**
-   * The key for a global property that defines one or more regular expressions
-   * that specify what should be included when searching for Stellar functions.
-   */
-  public static final String STELLAR_SEARCH_INCLUDES_KEY = "stellar.function.resolver.includes";
-
-  /**
+  public enum Config {
+    STELLAR_VFS_PATHS("stellar.function.paths", ""),
+    /**
+     * The key for a global property that defines one or more regular expressions
+     * that specify what should be included when searching for Stellar functions.
+     */
+    STELLAR_SEARCH_INCLUDES_KEY("stellar.function.resolver.includes", ""),
```

Comment:
If we leave it for that purpose, let's rename it and document it as such so it is clear.